js-client-examples/browser-example/package.json

```diff
@@ -51,8 +50,7 @@
         ]
     },
     "devDependencies": {
-        "@fluencelabs/cli": "0.9.1",
-        "@fluencelabs/aqua-lib": "0.7.7",
+        "@fluencelabs/cli": "0.12.1",
```

Comment:
I'm not sure we want people to do that, really. Better they install it through `install.sh`, and use `stable` channel.